.github/CODEOWNERS

```diff
@@ -127,7 +127,8 @@ app/controllers/v0/virtual_agent/report_to_cxdw.rb @department-of-veterans-affai
 app/controllers/v0/virtual_agent_token_controller.rb @department-of-veterans-affairs/vfs-virtual-agent-chatbot @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/controllers/v0/virtual_agent/virtual_agent_appeal_controller.rb @department-of-veterans-affairs/vfs-virtual-agent-chatbot @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/controllers/v0/virtual_agent/virtual_agent_claim_controller.rb @department-of-veterans-affairs/vfs-virtual-agent-chatbot @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/controllers/v1 @department-of-veterans-affairs/lighthouse-banana-peels
+app/controllers/v1 @department-of-veterans-affairs/backend-review-group
```

Comment:
I'm switching `controllers/v1` over to backend review, as there are both lighthouse and non-lighthouse stuff in there